data/notices.json

```diff
@@ -423,6 +423,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(aws-events): EventBus fails because of change to logical id without modifiying the statement id",
```

Comment:
this is what everyone who uses EventBus is going to see. Is it true that all users will see a failure? Because that is what this title is implying to me.